src/main/proto/marathon.proto

```diff
@@ -103,6 +103,7 @@ message ServiceDefinition {
   repeated mesos.Port portDefinitions = 27;
   repeated ReadinessCheckDefinition readinessCheckDefinition = 28;
   repeated Secret secrets = 29; // since 1.2
+  repeated EnvVarReference envVarReferences = 30;
```

Comment:
Maybe add: `// store env var reference separately as long as the Mesos CommandInfo cannot store these natively`